src/content/mission/mission.json

```diff
@@ -1,17 +1,17 @@
 {
-  "title": "Ethics as a Mission",
+  "title": "Our Core Values",
   "principles": [
     {
-      "title": "Empowerment",
-      "body": "We're ready to get serious with this. Just let it work around and play. Everybody should paint what they see and what they feel."
+      "title": "Teamwork",
+      "body": "We firmly believe that together, we can achieve more. Working in teams allows us to pair program and do thorough code reviews to ensure that we are delivering cleaner and more reliable solutions faster than any one of us could on our own."
```

Comment:
Can we get quality in here somewhere too, maybe replace "more reliable" with quality?

Reply:
I think reliable is more specific and will give us more credibility